tex/artikel/Gleichstellungsbuero.tex

```diff
@@ -1,35 +1,35 @@
-\begin{center}
-    \includegraphics[width=\columnwidth]{res/gst_QR.png} %Newsletter QR-Code
-\end{center}
+Beim Gender Breakfast Talk starten wir immer mit einem wissenschaftlichen Input-Talk Gleichstellungmit neusten Ergebnissen aus der Gender \& MINT Forschung. Dazu laden wir eine Rednerin ein, die in diesem Gebiet forscht. So bekommt jede:r Zuhörende die Möglichkeit, die eigenen Gender-Kompetenzen auszubauen – keine Angst, Vorkenntnisse werden nicht benötigt. Wichtig ist uns dabei, dass der Talk wissenschaftlich ist. Das bedeutet, dass alle Aussagen auf Studien beruhen und fundiert sind. Dementsprechend wollen wir eine offene und vielseitige Diskussion führen, was auch bedeutet, dass wir uns kritisch hinterfragen. Danach sprechen wir über das Thema des Vortrags oder auch jedes andere Thema, das aufkommt, klären Fragen und teilen unsere Erfahrungen in bestimmten Situationen. Dazu gibt es auch noch ein kleines kostenloses Frühstück.
 
-Grundsätzlich gilt: Wir sind die ersten Ansprechpersonen, wenn es um Gleichstellungsthemen geht. Also ruft uns einfach an (251 83-33516), schreibt uns eine E-Mail (\textbf{\email{gleichstellung.physik@uni-muenster.de}}) oder kommt auf einen Kaffee rum (Raum: AP106) und wir helfen euch. \\
+Das Physikerinnen-Café findet immer statt, wenn im Allgemeinen Physikalischen Kolloquium eine Rednerin spricht. Am selben Tag ihres Vortrags treffen wir uns vorab und sprechen sowohl über ihren Karriereweg als auch über geschlechterbezogene Probleme, denen sie während ihres eigenen Werdegangs begegnet ist. Meistens haben diese schon weiter fortgeschrittenen Wissenschaftlerinnen super Tipps für uns Studentinnen.
 
+Auch für die Themen sexualisierte Diskriminierung, Belästigung und Gewalt sind wir zuständig. Dabei haben wir immer ein offenes Ohr für euch, gehen mit euren Sorgen sowie Nöten diskret um und kennen die nötigen Ansprechpersonen, wenn weitere Schritte eingeleitet werden sollen.
 
-\fibelsig{Anna Niemann und Janice Bode}
+\begin{multicols}{2}
+Alle Infos dafür findet ihr auf unserer \mbox{Website\footnotemark} oder bekommt ihr regelmäßig, indem ihr euch auf unseren entspannten Newsletter setzt. Dies geht ganz einfach über diesen QR-Code:
+\fibelimgtext{
+    \includegraphics[height=4cm]{res/gleichstellung_qr.pdf}\\ %Newsletter QR-Code
+}{\url{http://go.wwu.de/v-rti}} % https://listserv.uni-muenster.de/mailman/listinfo/physikerinnen-fb11
 \end{multicols}
+\footnotetext{{\url{www.uni-muenster.de/Physik/department/equality/}}}
+
+Grundsätzlich gilt: Wir sind die ersten Ansprechpersonen, wenn es um Gleichstellungsthemen geht. Also ruft uns einfach an (251 83-33516), schreibt uns eine E-Mail (\textbf{\email{gleichstellung.physik@uni-muenster.de}}) oder kommt auf einen Kaffee rum (Raum: AP106) und wir helfen euch.
```

Comment:
$+$49 251 83-33516

Reply:
Tatsache, das ist unüblich, ich habe Janice einmal angeschrieben, wie die die Telefonnummer formatiert haben möchten.

Reply:
Habe eine Rückmeldung bekommen und die umgesetzt, es ist auf eine "0" am Anfang der Nummer hinausgelaufen.

Reply:
Ah okay, interessant, dass sie _diese_ Variante gewählt hat .. 
Weil mit $+$49 [...] ist die Nummer ja damit auch direkt ,,internationalisiert'', was besonders fremdsprachliche Studis freuen dürfte^^'

Reply:
Ja, andererseits ist das mit der 0 glaube ich günstiger für Leute aus Deutschland und in jedem Fall wissen die meisten internationalen Studis, die einen deutschsprachigen Gleichstellungstext lesen, ja vmtl., wie deutsche Telefonnummern mit 0 am Anfang gewählt werden. In jedem Fall Danke für die Anmerkung mit der Nummer!